joint_limits:
  joint1:
    has_position_limits: true
    min_position: 150.0
    max_position: 235.0
    has_velocity_limits: true
    max_velocity: 30. #56.0
    has_acceleration_limits: true
    max_acceleration: 5.0
    has_jerk_limits: true
    max_jerk: 100.0
    has_effort_limits: false
    max_effort: 1.0
    soft_min_position: 155.0
    soft_max_position: 230.0
    k_position: 2
    k_velocity: 1